### PR TITLE
Amls 4916 2

### PR DESCRIPTION
**Pull request by ajchilts**
Accessibility changes for Supervision section.

## Related / Dependant PRs?

N/A

## How Has This Been Tested?

Local + Unit

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/views/include/forms2/textarea.scala.html [hunk 1]
```diff
@@ -27,15 +27,45 @@
...
+                    @Messages(heading)
+                </label>
+            </h1>
+            @if(section.nonEmpty) {
+                <p class="heading-secondary">
+                    <span class="visuallyhidden">This section is: </span>@Messages(section)
+                </p>
+            }
+        </div>
+        <span class="form-label">
```
Comment (drewmiley) [inline app/views/include/forms2/textarea.scala.html]: <p>@labelSpan</p>

This is my fault, and is what Martin mentioned in the session earlier

File: app/views/supervision/another_body.scala.html [hunk 2]
```diff
@@ -36,39 +35,32 @@
...
-                target = s"#$target",
-                inline = true
+                target = s"#$target"
             )
             @panel(
                 field = Some(f(s"$target"))
             ) {
-
-                @errorGroup(
-                    field = f("supervisor-panel"),
```
Comment (drewmiley) [inline app/views/supervision/another_body.scala.html]: Im a little worried about the name of this removed errorGroup field. Do errors still work as expected on this page? If not, we'll take a look :)

File: app/views/supervision/member_of_professional_body.scala.html [hunk 3]
```diff
@@ -40,25 +39,23 @@
...
 
-        @fieldset(
-            legend = "supervision.memberofprofessionalbody.title",
-            legendHidden = true,
+        @fieldsetWithErrorGroup(
             f = f("isAMember"),
             panel = false,
-            classes = Seq("inline")
+            classes = Seq("inline"),
+            heading="supervision.memberofprofessionalbody.title",
```
Comment (drewmiley) [inline app/views/supervision/member_of_professional_body.scala.html]: Spacing

File: app/views/supervision/penalised_by_professional.scala.html [hunk 4]
```diff
@@ -27,50 +27,49 @@
 @header = {
     @components.back_link()
     @errorSummary(f)
-    @heading("supervision.penalisedbyprofessional.title", "summary.supervision")
+}
+
+@supportingContent = {
+<p>@Messages("supervision.penalisedbyprofessional.subtitle")</p>
```
Comment (drewmiley) [inline app/views/supervision/penalised_by_professional.scala.html]: Indent?

File: app/views/supervision/penalised_by_professional.scala.html [hunk 5]
```diff
@@ -27,50 +27,49 @@
...
     @form(f, controllers.supervision.routes.PenalisedByProfessionalController.post(edit)) {
 
-            @fieldset(
-                legend = "supervision.penalisedbyprofessional.title",
-                legendHidden = true,
+            @fieldsetWithErrorGroup(
                 f = f("penalised"),
-                panel = false
+                panel = false,
+                heading="supervision.penalisedbyprofessional.title",
```
Comment (drewmiley) [inline app/views/supervision/penalised_by_professional.scala.html]: Spacing

File: app/views/supervision/penalised_by_professional.scala.html [hunk 6]
```diff
@@ -27,50 +27,49 @@
...
                 )
-                @fieldset(
-                    legend = "supervision.penalisedbyprofessional.details.lbl",
-                    f = f(target),
-                    legendHidden = true
-                ) {
-                    @textarea(f("professionalBody"), "supervision.penalisedbyprofessional.details.lbl", bold = false, largeWidth = true, cols = "30", rows = "8")
-                }
+
+                @textarea(f("professionalBody"), "supervision.penalisedbyprofessional.details.lbl", bold = false, largeWidth = true, cols = "30", rows = "8")
```
Comment (drewmiley) [inline app/views/supervision/penalised_by_professional.scala.html]: Maybe put this on new lines

File: app/views/supervision/penalised_by_professional.scala.html [hunk 7]
```diff
@@ -27,50 +27,49 @@
...
+                classes = Seq("inline")
             ) {
+
                 @radio(
                     f = f("penalised"),
                     labelText = "lbl.yes",
                     value = "true",
-                    target = s"#$target",
-                    inline = true
+                    target = s"#$target"
```
Comment (drewmiley) [inline app/views/supervision/penalised_by_professional.scala.html]: Have you checked this conditional reveal works?

File: app/views/supervision/which_professional_body.scala.html [hunk 8]
```diff
@@ -36,13 +35,13 @@
...
     @form(f, controllers.supervision.routes.WhichProfessionalBodyController.post(edit)) {
 
-        @fieldset(
-            legend = "supervision.memberofprofessionalbody.title",
-            legendHidden = true,
+        @fieldsetWithErrorGroup(
             f = f("businessType"),
-            panel = false
+            panel = false,
+            heading="supervision.whichprofessionalbody.title",
```
Comment (drewmiley) [inline app/views/supervision/which_professional_body.scala.html]: Spacing

File: app/views/supervision/another_body.scala.html (hunk 2 above)
Comment (ajchilts) [reply app/views/supervision/another_body.scala.html]: Seems to work fine.

**Comment by ajchilts** [comment]
@drewmiley  - comments addressed.

**Pull request merged**